tests/p2p_stress.py

```diff
@@ -2,46 +2,94 @@
+            s=s+random.choice("abcdefghijklmnopqrstuvwxyz12345")
+        return s
+    
+    def _transfer(self, node, acc1, acc2, amount, threadId, round):
+        memo="%d %d" % (threadId, round)
+        tr = node.transferFunds(acc1, acc2, amount, memo)
+        self.trList.append(tr)
+
     def execute(self, cmdInd, node, ta, eosio):
-        print("\n==== stress network test: %d transaction(s)/s for %d secs ====" % (self.speeds[cmdInd], self.sec))
+        print("\n==== network stress test: %d transaction(s)/s for %d secs ====" % (self.speeds[cmdInd], self.sec))
         total = self.speeds[cmdInd] * self.sec
-        acclist = []
-        for k in range(total):
-            s=""
-            for i in range(10):
-                s=s+random.choice("abcdefghijklmnopqrstuvwxyz12345")
-            acclist.append(s)
-        ta.name = s
-        transIdlist = []
-        print("==== creating %d account(s) ====" % (total))
-        delay = 1.0 / self.speeds[cmdInd]
+
+        ta.name = self.randAcctName()
+        acc1 = copy.copy(ta)
+        print("creating new account %s" % (ta.name))
+        tr = node.createAccount(ta, eosio, stakedDeposit=0, waitForTransBlock=False)
+        print("transaction id %s" % (node.getTransId(tr)))
+
+        ta.name = self.randAcctName()
+        acc2 = copy.copy(ta)
+        print("creating new account %s" % (ta.name))
+        tr = node.createAccount(ta, eosio, stakedDeposit=0, waitForTransBlock=False)
+        print("transaction id %s" % (node.getTransId(tr)))
+
+        time.sleep(1.0)
+
+        print("issue currency into %s" % (acc1.name))
+        contract="eosio"
+        action="issue"
+        data="{\"to\":\"" + acc1.name + "\",\"quantity\":\"1000000.0000 EOS\"}"
+        opts="--permission eosio@active"
+        tr=node.pushMessage(contract, action, data, opts)
+        print("transaction id %s" % (node.getTransId(tr[1])))
+
+        time.sleep(1.0)
+
+        self.trList = []
+        expBal = 0
+        nthreads=self.maxthreads
+        if nthreads > self.speeds[cmdInd]:
+            nthreads = self.speeds[cmdInd]
+        cycle = int(total / nthreads)
+        total = cycle * nthreads # rounding
+        delay = 1.0 / self.speeds[cmdInd] * nthreads
+
+        print("start currency trasfer from %s to %s for %d times with %d threads" % (acc1.name, acc2.name, total, nthreads))
+
         t00 = time.time()
-        t0 = time.time()
-        trList = []
-        for k in range(total):
-            ta.name = acclist[k]
-            tr = node.createAccount(ta, eosio, stakedDeposit=0, waitForTransBlock=False)
-            trList.append(tr)
+        for k in range(cycle):
+            t0 = time.time()
+            amount = 1
+            threadList = []
+            for m in range(nthreads):
+                th = threading.Thread(target = self._transfer,args = (node, acc1, acc2, amount, m, k))
+                th.start()
+                threadList.append(th)
+            for m in range(len(threadList)):
```

Comment:
Python idiom is

```
for thread in threadList:
   thread.join()
```
No need to index into the list.

This comment applies throughout the code.